src/templates/app/cashier/payment_methods.jsx

```diff
@@ -563,8 +564,8 @@ const PaymentMethods = () => {
                                         { td: it.L('Tether is a blockchain-based cryptocurrency whose cryptocoins in circulation are backed by an equivalent amount of traditional fiat currencies. For more information, please visit [_1].', `${createLink('https://www.tether.to')}`) },
                                         { td_list: [
                                             { text: 'UST' },
-                                            { text: '1' },
-                                            { text: '1' },
+                                            { text: '-' },
+                                            { text: '-' },
```

Comment:
instead of `-` let's leave the withdrawals empty? since they will be populated after response comes, so that user doesn't think it's not available